generator/src/link.ts

```diff
@@ -0,0 +1,78 @@
+/**********************************************************************
+ * Copyright (c) 2018-2020 Red Hat, Inc.
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ ***********************************************************************/
+
+/**
+ * A command to yarn link theia dependencies into che-theia
+ * @author Thomas Mäder
+ */
+
+import * as fsextra from 'fs-extra';
```

Comment:
why not only `fs` ?

Reply:
Conversely, why import fs-extra as 'fs'?

Reply:
because it's a drop-in replacement (and all over theia source code it's currently imported as 'fs')

Reply:
As I understand it, it has more API than fs, so I wouldn't call it a drop-in replacement, but the "convention" argument is fair enough.

Reply:
it's how it is advertised
https://github.com/jprichardson/node-fs-extra#usage

> fs-extra is a drop in replacement for native fs